### PR TITLE
Closes #2491: Re-exported configuration files for 2.7.x

### LINKED ISSUE #2491: Re-export all config before Quickstart 2.7.0 stable release

### PR DESCRIPTION
**Pull request by joeparsons**
## Description
Re-exported all config using @tadean's sweet new drush command.  Tried to capture only actual configuration drift changes.  Omitted optional/override changes related to metatag, multilingual, etc.

## Related issues

Closes #2491 

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [x] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_event/az_event_trellis/config/install/migrate_plus.migration.az_trellis_events.yml [hunk 1]
```diff
@@ -29,54 +35,48 @@ source:
...
     -
       plugin: skip_on_empty
       method: process
       source: summary
-
-  # Events are considered "published" unless any of these conditions are met:
-  # - 'External_Visibility' is "Don't Broadcast to any Platform (invite only)"
-  # - 'Status_Formula' is "Canceled"
-  # = 'Publish' is "false"
   external_visibility:
```
Comment (joeparsons) [inline modules/custom/az_event/az_event_trellis/config/install/migrate_plus.migration.az_trellis_events.yml:80]: I think we should preserve these comments.
```suggestion

  # Events are considered "published" unless any of these conditions are met:
  # - 'External_Visibility' is "Don't Broadcast to any Platform (invite only)"
  # - 'Status_Formula' is "Canceled"
  # = 'Publish' is "false"
  external_visibility:
```

File: modules/custom/az_media/config/install/core.entity_form_display.media.az_audio.default.yml [hunk 2]
```diff
@@ -50,3 +50,4 @@ content:
     third_party_settings: {  }
 hidden:
   name: true
+  replace_file: true
```
Comment (trackleft) [inline modules/custom/az_media/config/install/core.entity_form_display.media.az_audio.default.yml:53]: This comes with media_entity_file_replace, right?

File: modules/custom/az_flexible_page/config/install/core.entity_view_display.node.az_flexible_page.default.yml [hunk 3]
```diff
@@ -35,3 +35,4 @@ hidden:
   field_az_media_image: true
   field_az_page_category: true
   field_az_summary: true
+  smart_title: true
```
Comment (trackleft) [inline modules/custom/az_flexible_page/config/install/core.entity_view_display.node.az_flexible_page.default.yml:38]: I think this is correct despite https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_core/config/install/smart_title.settings.yml#L6

Especially considering https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/config/optional/block.block.az_barrio_page_title.yml#L23

File: config/install/block.block.az_barrio_main_menu.yml [hunk 4]
```diff
@@ -4,7 +4,6 @@ dependencies:
   config:
     - system.menu.main
   module:
-    - block_class
```
Comment (trackleft) [inline config/install/block.block.az_barrio_main_menu.yml:7]: We should test these on blocks with and without classes added.

Comment (joeparsons) [reply config/install/block.block.az_barrio_main_menu.yml:7]: Done ✅

**Review by trackleft — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**